monai/transforms/post/contour.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn as nn
+import numpy as np
+
+class Contour:
```

Comment:
1. Could you please move this transform into `monai.transforms.post.array.py` and follow the same pattern as other transforms?
2. Maybe change the class name to `KeepContour` or `GetContour`?

Thanks.

Reply:
1. sure, already done
2. I don't think `GetContour` is a good class name, it's more like method name.

Reply:
OK, let's keep the name.